### ISSUE TITLE
lua

### ISSUE THREAD

**Issue by swobodac** [ShadowMario/FNF-PsychEngine#10560]
### Describe your problem here.

do you know how to randomise the bpm like unfairness?

### Are you modding a build from source or with Lua?

Lua

### What is your build target?

Windows x64

### Did you edit anything in this build? If so, mention or summarize your changes.

os engine lol

**Comment by Isaagomen** [comment]
ok first,
1 - Why, for a fucking d&b mod?
2 - This is a psych issue chat, not os engine.

**Comment by swobodac** [comment]
1 - im making a port
2 - os engine is pe but modified

**Comment by swobodac** [comment]
3- os engine is read only

**Comment by swobodac** [comment]
anybody?
hello?

**Comment by DisIsNOt** [comment]
math.random

Go find yourself for how to use it

**Comment by Lorekeeper49** [comment]
`getRandomInt(min, max, exclude?)

**Comment by BernardoGP4504** [comment]
> 3- os engine is read only

yes but just because of that it doesn't mean that you can post the issue on the origin of the engine
so try to post there or in their discord server idk

**Comment by swobodac** [comment]
i cant use discord.

**Comment by BernardoGP4504** [comment]
> i cant use discord.

Oh ok also you change the bpm in a section on the chart and then it works for the rest
But idk how to randomize the bpm

**Comment by swobodac** [comment]
> > i cant use discord.
> 
> Oh ok also you change the bpm in a section on the chart and then it works for the rest But idk how to randomize the bpm

read the tittle
